### ISSUE TITLE
Version 6.7.2 seems not to transpile to es5

### ISSUE THREAD

**Issue by Stoobaart** [ember-cli/ember-fetch#435]
Not sure what's causing this bug, but it appears that ember-fetch is not transpiling to es5 on version 6.7.2

Using Ember-cli 3.15.1
Confirmed using es-check

Reproduction steps:
In terminal:
npm i es-check -g 
Using an Ember app with ember fetch version 6.7.2...
npm i
ember build --environment production
es-check es5 './dist/**/*.js'

Expected result: ES-Check: there were no ES version matching errors!

Actual result: ES-Check: there were 1 ES version matching errors.
          · erroring file: ./dist/ember-fetch/fetch-fastboot-38cfd9007f94f81f5a2bc13690efc343.js
          · error: SyntaxError: The keyword 'const' is reserved (3:0)

We get the expected result with version 6.7.0

**Comment by xg-wang** [comment]
`const` from `fetch-fastboot.js` is from this line https://github.com/ember-cli/ember-fetch/blob/v6.7.2/public/fetch-fastboot.js#L16.
The file only runs in Fastboot(node). And `const` is available in pretty much all nodejs versions.
Does other dis/**/*.js files pass the test?

**Comment by Stoobaart** [comment]
Yes, all other packages pass, this is the only one that fails. I'm not sure at all what's causing the issue, but it went away when I down graded to 6.7.0, thought it would be best to let someone know :)
